### DISCUSSION TITLE
关于docker启动，显示代理无法使用问题

### DISCUSSION THREAD

**Discussion by MuXia-0326** [1715173329/UnblockNeteaseMusic#413] [Q&A]
docker run -e ENABLE_FLAC=true -e MIN_BR=320000 -d -p 12000:8080 pan93412/unblock-netease-music-enhanced -p 8080:8081 -o kuwo kugou bilibili -e http://music.163.com -f （云服务器ip）

用上面的命令启动，并开启了12000的防火墙，然后用网易云是测试显示
![image](https://user-images.githubusercontent.com/63834850/147426643-f2ec95d3-b379-4582-8cb4-b1469efe5fc6.png)

**Comment by 1715173329** [comment]
这是正常现象。设定代理后能正常通讯即可。

**Reply by MuXia-0326** [reply]
> 

但是我设置代理后，网易云也是无法使用的情况

**Reply by MuXia-0326** [reply]
捣鼓了一阵，启动命令没看出啥问题，ping docker的端口也是可以ping通的

**Comment by spicy-man** [comment]
我也是遇到了这个问题，还没能找到解决方法。
使用本地的192.168.xx.xx加端口可以，映射到公网就不能用。
